code/datums/components/material/material_container.dm

```diff
@@ -66,6 +66,9 @@
 		for(var/signal in container_signals)
 			parent.RegisterSignal(src, signal, container_signals[signal])
 
+	//drop sheets when the object is deconstructed but not deleted
+	RegisterSignal(parent, COMSIG_OBJ_DECONSTRUCT, TYPE_PROC_REF(/datum/component/material_container, drop_sheets))
```

Comment:
Why is this a `TYPE_PROC_REF` and not just a `PROC_REF`

Reply:
No reason, just a habit i follow if the proc is attached to an type. Is there any benifit to using normal `PROC_REF` over this?